src/components/browse/sidebar/sidebar.tsx

```diff
@@ -5,43 +5,56 @@ import { Row, Col, Button } from "react-bootstrap";
 import { facetModel, facetFilterModel } from "../../../models/facets";
 import { searchResponseModel } from "../../../models/dataset";
 import { getDatasetsSearchResp } from "../../../api/browse";
+import PerfectScrollbar from 'react-perfect-scrollbar'
+import 'react-perfect-scrollbar/dist/css/styles.css';
 
 interface sidebarProps {
   facetList: facetModel[] | null;
   setSearchResp: Dispatch<SetStateAction<searchResponseModel | null>>;
-  searchKeyword: string
-  limit: number
-  setSearchKeyword: Dispatch<SetStateAction<string>>
+  searchKeyword: string;
+  limit: number;
+  setSearchKeyword: Dispatch<SetStateAction<string>>;
   setFilterDict: Dispatch<SetStateAction<facetFilterModel[]>>;
   filterDict: facetFilterModel[];
 }
 
 const Sidebar = (props: sidebarProps) => {
-  const [check, setCheck] = useState<Map<string, boolean>>(new Map<string, boolean>());
+  const [check, setCheck] = useState<Map<string, boolean>>(
+    new Map<string, boolean>()
+  );
   const skip = 0;
   const handleClear = () => {
-    getDatasetsSearchResp(props.setSearchResp, [], '*', skip, props.limit);
+    getDatasetsSearchResp(props.setSearchResp, [], "*", skip, props.limit);
     check.forEach((value: boolean, key: string) => {
-      setCheck(check.set(key, false))
-    })
-    props.setFilterDict([])
+      setCheck(check.set(key, false));
+    });
+    props.setFilterDict([]);
   };
 
   const handleFilter = () => {
-    getDatasetsSearchResp(props.setSearchResp, props.filterDict, props.searchKeyword, skip, props.limit);
+    getDatasetsSearchResp(
+      props.setSearchResp,
+      props.filterDict,
+      props.searchKeyword,
+      skip,
+      props.limit
+    );
   };
 
   return (
     <div>
       <Row>
-        <Search searchKeyword={props.searchKeyword}
+        <Search
+          searchKeyword={props.searchKeyword}
           setSearchKeyword={props.setSearchKeyword}
           setSearchResp={props.setSearchResp}
-          limit={props.limit} />
+          limit={props.limit}
+        />
       </Row>
       {props.facetList === null || props.facetList.length < 1 ? null : (
-        <div className="bg-light border p-2 rounded-3">
-          <Row>
+        <div className="bg-light border p-2 rounded-3 pt-3">
+          <Row style={{height: "65vh"}} className="position-relative w-100 px-0 mx-0">
```

Comment:
will the hardcoded height work as intended on various screens of different size?